src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -236,7 +236,7 @@ to the `<a>` tag. On Android, links with `target="_blank"` will open in a
 A few other notes on `scope`:
 
 * If you don't include a `scope` in your manifest, then the default implied
-  `scope` is the directory that your web app manifest is served from.
+  `scope` is the parent directory of your `start_url`.
```

Comment:
I don't see this in the spec, maybe I'm missing something. Can you clarify?

I *thought* the scope would be the directory where the PWA is served... 

For example:
```
scope: '/pwa/',
start_url: '/pwa/start.html',
```

The scope would be `/pwa/` and everything below that, and when launched it would be launched from `/pwa/start.html`.

Reply:
The web app manifest can be served from anywhere, doesn't have to be the same origin.
The scope defaults to the parent directory of the start_url.

Spec: https://www.w3.org/TR/appmanifest/#scope-member
> Set manifest["scope"] to the result of parsing "." with manifest["start_url"] as the base URL.